app/_how-tos/route-requests-to-backup-targets.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Route requests to backup Targets during failures
+content_type: how_to
+
+description: Use the Route by Header plugin to route requests based on a header value.
+
+products:
+    - gateway
+
+works_on:
+    - on-prem
+
+min_version:
+  gateway: '3.12'
+
+entities: 
+  - service
+  - route
+  - upstream
+  - target
+
+tags:
+  - failover
+search_aliases:
+  - backup target
+
+tldr:
+    q: How do I route requests to different Targets in case my other Targets are unhealthy?
+    a: "Create a Service, a Route, and an Upstream with one of the `latency`, `least-connections`, or `round-robin` load balancing strategies. Configure primary Targets on the Upstream with `failover: false` and a failover Target with `failover: true`."
+
+tools:
+    - deck
+related_resources:
+  - text: Upstream entity
+    url: /gateway/entities/upstream/
+  - text: Target entity
+    url: /gateway/entities/target/
+
+cleanup:
+  inline:
+    - title: Destroy the {{site.base_gateway}} container
+      include_content: cleanup/products/gateway
+      icon_url: /assets/icons/gateway.svg
+---
```

Comment:
```suggestion

faqs:
  - q: Can I set more than one failover Target?
    a: Yes, Upstream support multiple failover Targets.
---
```
Something like this, to make sure that people don't think this tutorial is the only way.